src/main/java/service/impl/WriterServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package service.impl;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import service.WriterService;
+
+public class WriterServiceImpl implements WriterService {
+    @Override
+    public void writeToFile(String report, String toFileName) {
+        File file = new File(toFileName);
+        if (!file.exists()) {
+            try {
+                file.createNewFile();
+                Files.write(file.toPath(), report.getBytes(), StandardOpenOption.APPEND);
```

Comment:
`StandardOpenOption.APPEND` parameter is used when file is already exist and we need append data, but in `if` block in line 13 is checked that `file not exists`.